crates/ra_ide/src/call_info.rs

```diff
@@ -208,17 +208,18 @@ mod tests {
         }
     }
 
-    fn call_info(text: &str) -> CallInfo {
+    fn call_info(text: &str) -> Option<CallInfo> {
```

Comment:
It’s better to have two functions, call_info and no_call_info, which call the same helper and assert some/none